packages/runtime/container-runtime/README.md

```diff
@@ -51,6 +51,54 @@ issue](https://github.com/microsoft/FluidFramework/wiki/Submitting-Bugs-and-Feat
 
 Thank you!
 
+## Data Virtualization For DataStores
+
+This is the ability of container to delay load datastores at any point in time by asking for the datastore snapshot at
+anytime, not caring about from where it is coming and then using that to load that datastore and return it to the app.
+
+### How Current System Works
+
+Fluid (through its APIs) does expose some levels of data virtualization. But under covers, there is almost none. The
+whole content of the file is downloaded in one go. Due to limitation of data virtualization, FF holds ALL blobs in
+snapshot as those might be required in the future. Any delayed loading (through FF APIs) results in loading state of
+DDSs at a sequence number of snapshot we booted from, and application of all the ops up till current sequence number.
+While application may choose not to load some data stores immediately on boot (and realize some saving in time and
+memory by not allocating appropriate app state for such datastores), FF still pays the costs for such content.
+It also continues to pay the cost for all such content indefinitely, even if those datastores were loaded (this could
+theoretically be optimized, but now there is no good way for driver to learn what blobs are safe to discard due to
+limitations in interface and lack of appropriate semantics at driver API layer).
+
+### Improvement with Data Virtualization
+
+With this, we will provide a capability to:
+
+- Exclude some sub-trees from snapshot payload, thus allowing faster transfer times / boot times and smaller initial
+memory footprint.
+- Ability to safely (in current system, i.e., without introducing any new breaking behaviors) to delay-load data
+stores later.
```

Comment:
I'd not mention safety - it's assumed implicitly that system works correctly and without bugs

Reply:
done